django_socio_grpc/services/servicer_proxy.py

```diff
@@ -279,23 +282,41 @@ def __getattr__(self, action):
 
         return self.get_handler(action)
 
-    def process_exception(self, exc, request_container: GRPCRequestContainer):
-        if isinstance(exc, GRPCException):
-            request_container.context.abort(exc.status_code, exc.get_full_details())
+    def _get_status_code_and_details(self, exc: Exception) -> Tuple[grpc.StatusCode, str]:
```

Comment:
Need doc string for this method explaining why we need to convert from REST code to gRPC code. 

For reflexion why this is done in the servicer ? doesn't need self. Can be a method in exception.py, no ?